src/validate_pyproject/plugins/setuptools.schema.json

```diff
@@ -100,7 +100,7 @@
       "type": "object",
       "additionalProperties": false,
       "propertyNames": {
-        "oneOf": [{"const": ""}, {"$ref": "#/definitions/package-name"}]
+        "anyOf": [{"const": ""}, {"$ref": "#/definitions/package-name"}]
```

Comment:
Does SchemaStore prefers `anyOf` instead of `oneOf`?

Reply:
I see... in the case `format` is excluded, it starts to match multiple definitions and then fails.